### DISCUSSION TITLE
How to specify back in vue-router?

### DISCUSSION THREAD

**Discussion by yetrun** [vuejs/router#1637] [Help and Questions] [answered]
### What problem does this feature solve?
The vue-router can go to any route through specify `to` property, but how can it go back like below:

```javascript
<a @click="$router.back()"></a>
```

### What does the proposed API look like?
Does it exist a api like below:

```javascript
<vue-router back></vue-router>
```

**Comment by posva** [comment, marked as answer]
A back button should be a button because not only there isn't a reliable way to get the back url but also because it's not a link, it's an action that depends on the previous position and should be loaded within the document without allowing other actions like cmd+click:

```vue
<button @click="$router.back()">Go back</a>
```

That's also the reason this kind of behavior doesn't exist in `<RouterLink>`.

Maybe, in the
